vector-im/element-android
# [Devices management] Add a feature flag and empty screen for future new layout (PSG-676)

## Type of change


## Content


Adding a new feature flag to start new layout for device management in settings.

## Motivation and context


Closes #6798 

## Screenshots / GIFs




When feature flag is enabled:

<img src="https://user-images.githubusercontent.com/46314705/183858520-109dbffe-1110-47eb-a884-aea66b95e79a.png" width=300 /> <img src="https://user-images.githubusercontent.com/46314705/183858513-3bf44a28-a67e-4f27-b735-7fa7cf555486.png" width=300 />

## Tests



- Go to settings -> Security & Privacy
- Check in the "Active Sessions" section there is only one entry (the current one)
- Go to debug settings -> Features
- Enable the new device management feature flag
- Go to settings -> Security & Privacy
- Check in the "Active Sessions" section there is a new entry below the current one
- Press this new entry
- Check an empty screen with title "Sessions" is displayed

## Tested devices

- OS version(s): Android 11

## Checklist
